doc/nrf/protocols/thread/device_types.rst

```diff
@@ -95,31 +101,36 @@ Minimal Thread Devices can be further divided into three categories:
      - No
      - No
      - No
-   * - Radio on
-     - Always
-     - Periodically
-     - Periodically
+   * - Radio
+     - Always enabled
+     - Periodically enabled
+     - Periodically enabled
    * - New messages
      - Parent immediately forwards the messages
      - The SED polls for new messages when it wakes up
-     - Parent forwards the messages during designated transmission window
+     - Parent forwards the messages during designated transmission window or when SSED explicitly polls data
    * - No new messages
      - No data transmission
      - Parent indicates no pending messages
-     - No data transmission
+     - No data transmission*
 
-MEDs are the most basic MTDs, and their radio is always on.
-They don't keep a routing table, but otherwise operate like :ref:`FEDs <thread_types_ftd>`.
+MEDs are the most basic MTDs, and their radio is always enabled.
+They don't keep a routing table and have a :ref:`more limited range of functionalities than FEDs <thread_types_ftd>`.
 
 SEDs try to limit their power consumption by sleeping most of the time, waking up periodically to poll for messages from their parent.
 After waking up, they send a data request to their parent.
 If the parent has any pending messages, it will send them to the SED.
+(This can happen at any time, since Routers have their radio enabled all the time.)
```

Comment:
nit: Does it make sence to have this information together with previous sentence? I mean...

"[..] to the SED (this can happen at any time..."